app/containers/Communities/index.js

```diff
@@ -78,12 +79,14 @@ export const Communities = ({
 
   return (
     <div className="d-xl-flex">
-      <Seo
-        title={translationMessages[locale][messages.title.id]}
-        description={translationMessages[locale][messages.description.id]}
-        language={locale}
-        keywords={keywords}
-      />
+      {process.env.ENV !== 'dev' && (
```

Comment:
What is the reason for not rendering Seo elements in DEV?

Reply:
This is a workaround that was provided by @GoodPassiveMan to avoid errors that Seo component generates on dev-environment.